sandbox/modules/identity/src/internalClusterTest/java/org/opensearch/identity/SecurityPluginCompatibilityIT.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Does this file name stand for SecurityPluginCompatabilityIntegrationTest? If so is there a character limit or standardized format that makes us want to abbreviate the name to end with 'IT'? If not I would suggest using the longer version considering we do have other <x>IntegrationTest.java files and IT is ambiguous.

Reply:
Ending an integration test in `IT` is the convention. There are a few gradle files out there that rely on this to find the tests to perform.

Reply:
Good to know, I did not realize we had IT as the convetion--for some reason I thought it looked for Integ or Integration. But sounds good then.